lms/djangoapps/mobile_api/users/views.py

```diff
@@ -70,27 +69,35 @@ class UserDetail(generics.RetrieveAPIView):
 @mobile_view(is_user=True)
 class UserCourseStatus(views.APIView):
     """
-    **Use Case**
+    **Use Cases**
 
         Get or update the ID of the module that the specified user last visited in the specified course.
```

Comment:
@nasthagiri In the documentation for this endpoint, this is described as "Get or change a user's status in a course." Is getting or patching the last visited module ID the same as getting or changing a user's status? Can we re-word either this string or the description in the documentation so that they look more similar?